tdrs-backend/tdpservice/stts/test/test_models.py

```diff
@@ -1,7 +1,7 @@
 """Module for testing the stt and region models."""
 import pytest
 
-from ..models import STT, Region
+from tdpservice.stts.models import STT, Region
```

Comment:
Why was this change made? Does the import fail otherwise?

Reply:
Yes, it was failing on my local. 
Besides PEP8 discourages relative imports and encourages explicit imports: "Relative imports for intra-package imports are highly discouraged. Always use the absolute package path for all imports." For django, importing form the same directory using one dot is standard.